src/address_book.py

```diff
@@ -155,10 +182,45 @@ def add_address(self, address: str) -> None:
         """
         self.address = address
 
+    def add_email(self, email):
+        """Add an email to the contact.
+
+        :param email: The email to add.
+        """
+        self.emails.append(Email(email))
+
+    def edit_email(self, old_email, new_email):
+        """Edit the email for a contact.
+
+        :params: old_email, new_email.
+        :return: str: Result message.
+        """
+        if self.emails:
```

Comment:
This code can be improved 
```
        old_email_obj = Email(old_email)
        if old_email_obj in self.emails:
            self.emails[self.emails.index(old_email_obj)] = Email(new_email)
        else:
            raise HelperError(f"Email '{old_email}' doesn't exist for this contact.")
```
class Field has magic method `__eq__` this allows us to check if object itself in the list using `in`